Assets/MXR.SDK/Runtime/Types/ContentTypes.cs

```diff
@@ -208,6 +208,11 @@ public class RuntimeApp : Content {
         /// </summary>
         public string packageName;
 
+        /// <summary>
+        /// Whether the application has expired or not
```

Comment:
This comment mostly just reiterates the name of the field. Can you make this clear to someone with less context who is looking at our SDK?

Maybe something like: 

"Whether the application is currently in an expired state, which happens on shared applications when they reach the end of their shared time limit."